application-github-statistics-ui/src/main/resources/GitHubStats/CommittersMacro.xml

```diff
@@ -196,13 +385,13 @@
   #if ($isCommitter || (!$isCommitter &amp;&amp; "$!xcontext.macro.params.contributors" == "true"))
     #set ($avatar = $authorData.avatar)
     #if ("$!avatar" == '')
-      #set ($avatarDisplay = "(% style='width:85px' %)[[image:attach:GitHubStats.CommittersMacro@unknown.png||width='80' height='80']]")
+      #set ($avatarImg = "attach:GitHubStats.CommittersMacro@unknown.png")
     #else
-      #set ($avatarDisplay = "(% style='width:85px' %)[[image:${avatar}||width='80' height='80']]")
+      #set ($avatarImg = "${avatar}")
     #end
+    #set ($avatarDisplay = "[[image:${avatarImg}]]")
     #set ($company = $authorData.company)
     #if ("$!company" != '')
-      #set ($companyDisplay = "\\//{{{$company}}}//")
```

Comment:
Missed it in the commit, added it back in 5504d575918e02b2cd9e718b71159c4a523c74e0